const mergeError = error => ({ error, status: 500 });

export default mergeError;
